### PR TITLE
Adjust the position of zombified piglin on strider.

### PR DESCRIPTION
**Pull request by HeyDon-Lee**

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java [hunk 1]
```diff
@@ -141,7 +141,17 @@ private void updateOffset(Entity passenger, EntityType mountType, GeyserSession
                 yOffset = 1.3f;
                 break;
             case STRIDER:
-                yOffset = passenger.getEntityType() == EntityType.PLAYER ? 2.8200102f : 1.6f;
+                switch(passenger.getEntityType()) {
+                    case EntityType.SPIDER:
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java:145]: I assume you meant `EntityType.STRIDER` here.

**Comment by DoctorMacc** [comment]
I'm trying to look and see if there's some equation that gets the correct height. If I am unsuccessful, I will use this PR.

File: connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java (hunk 1 above)
Comment (HeyDon-Lee) [reply connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java]: I'm sorry.
I pressed too fast and missed a button.

Comment (DoctorMacc) [reply connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java]: There is absolutely no need to apologize! The point of us reviewing is to catch those small mistakes.

**Comment by DoctorMacc** [comment]
I didn't get too far in my mount offset quest; rebase/reopen this to the Master branch and I will approve it.

**Pull request closed without merging**